subtrack.MAUI/Pages/Subscriptions.razor

```diff
@@ -0,0 +1,11 @@
+﻿@using Components
+@page "/subscriptions"
+
+<h3>Subscriptions</h3>
+
+<div style="
```

Comment:
place this inside the button component instead. also add it to the startpage

Reply:
Moved the CSS as requested and added a z-index.